module/documents/chat-message.mjs

```diff
@@ -55,6 +55,20 @@ export default class CrucibleChatMessage extends ChatMessage {
     await CrucibleAction.confirmMessage(this, {action});
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Returns the most recent action (confirmed or otherwise) in the chat log
+   * @returns {CrucibleAction|null}
+   */
+  static getLastAction() {
```

Comment:
Good start, I suggest adding some parameter options to this. some possibilities include:
* `{boolean} [confirmed]` - require the action to be confirmed (true) unconfirmed (false) or either (undefined)
* `{string|Actor} [actor]` - An Actor (or actor ID) who was the person to perform the action

Maybe others in the future? Signature like:
```suggestion
  static getLastAction({confirmed, actor}={}) {
```